### ISSUE TITLE
Can't procced to backward.loss process

### ISSUE THREAD

**Issue by ganeshkumartk** [facebookresearch/inversecooking#8]
Found a bug in line 390

 `from src/modules/transformer_decoder.py
logits = torch.stack(logits, 1).data --> logits = torch.stack(logits, 1)`

As such, backward.loss process without the require_grad being True, creates error in training.

**Comment by amaiasalvador** [comment]
Duplicate of #3 . Fixed by 3ab629c08f0b5d84bbc3d56ac70ce12e60500322.
